contracts/libraries/IterableAppendOnlySet.sol

```diff
@@ -0,0 +1,34 @@
+pragma solidity ^0.5.0;
+
+library IterableAppendOnlySet {
```

Comment:
You are storing addresses, I would reflect that in the name, since `IterableAppendOnlySet` is too general